src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -31,20 +33,22 @@ public class AddCommandParser implements Parser<AddCommand> {
      */
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_TAG);
+                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_TELEGRAM,
+                        PREFIX_GITHUB, PREFIX_TAG);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ADDRESS, PREFIX_PHONE, PREFIX_EMAIL)
-                || !argMultimap.getPreamble().isEmpty()) {
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_TELEGRAM, PREFIX_PHONE,
+                PREFIX_EMAIL, PREFIX_GITHUB, PREFIX_TAG) || !argMultimap.getPreamble().isEmpty()) {
```

Comment:
I think checking the `PREFIX_TAG` is not needed here? I think the user does not need to tag every person he/she inputs.

That would pass the test cases in `LogicManagerTest`